src/components/TagMenu/index.web.tsx

```diff
@@ -66,11 +66,9 @@ export function TagMenu({
         !isInvalidHandle(authorHandle) && {
           label: _(msg`See ${truncatedTag} posts by user`),
           onPress() {
-            navigation.navigate({
-              name: 'Search',
-              params: {
-                q: tag + (authorHandle ? ` from:${authorHandle}` : ''),
-              },
+            navigation.push('Hashtag', {
+              tag: tag.replace('#', ''),
+              author: authorHandle,
```

Comment:
If we add the '#' into the URL, it gets removed by the router, so the route just ends up being `/hashtag` on web...I wanted to do this too but alas :/

Reply:
Should you be using encodeURIComponent here? What if it's the ##hashtag case?

Reply:
@estrattonbailey What's your take on that now that we've changed the way they get sanitized?

Reply:
Yep `##hashtag` is still valid, so gotta encode. You're right though, the URLs would look kinda jank for "normal" tags with just one hash 🤔 

I'm actually wrong, what gets passed to `TagMenu` is the _text_ portion of the tag. We should really pass in the `tag` itself, since that's already sanitized. Then we know there's no leading # (unless it's a double hash like `##hashtag`), and yeah we need to encode for those cases. That make sense?

Reply:
Okay this logic should work now, leaving a comment below.